notebooks/official/training/xgboost_data_parallel_training_on_cpu_using_dask.ipynb

```diff
@@ -61,7 +61,7 @@
       "source": [
```

Comment:
<p>us-docker.pkg.dev/vertex-ai/prediction/tf2-cpu.2-8:latest" - I recommend define this as DEPLOY_IMAGE</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1489/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>